kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/quectel/ec25/QuectelEC25AtCommands.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
```

Comment:
Please see the comments of the other class similar to this

Reply:
ok